pallets/marketplace/src/lib.rs

```diff
@@ -590,6 +662,10 @@ pub mod pallet {
         MarketplaceUriUpdated(MarketplaceId, URI),
         /// Marketplace Logo URI updated. \[marketplace id, logo URI\]
         MarketplaceLogoUriUpdated(MarketplaceId, URI),
+        /// Account added to disallow list for a marketplace.  \[marketplace id, account\]
+        AccountBlacklistedFromMarketplace(MarketplaceId, T::AccountId),
```

Comment:
Because of the recent trends in the IT industry terms like whitelist and blacklist are not anymore welcome. I suggest we rename the existing events `AccountAddedToMarketplace` and `AccountRemovedFromMarketplace`  to `AccountAddedToAllowList` and `AccountRemovedFromAllowList`, respectively. With that change we would be able to name the thew new events in the same manner, so we would have  `AccountAddedToDisallowList` and `AccountRemovedFromDisallowList`. What do you think?

Reply:
Yes make sense